### PR TITLE
Add telemetry aggregates parquet job (bug 1475358)

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: dags/telemetry_aggregates_parquet.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    'email': ['telemetry-alerts@mozilla.com', 'robhudson@mozilla.com', 'frank@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 3,
+    'retry_delay': timedelta(minutes=30),
+}
+
+dag = DAG('telemetry_aggregates_parquet', default_args=default_args, schedule_interval='@daily')
+
+telemetry_aggregate_parquet_view = EMRSparkOperator(
```
Comment (fbertsch) [inline dags/telemetry_aggregates_parquet.py]: Want to run this on databricks?

File: dags/telemetry_aggregates_parquet.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    task_id="telemetry_aggregate_parquet_view",
+    job_name="Telemetry Aggregate Parquet View",
+    instance_count=40,
```
Comment (fbertsch) [inline dags/telemetry_aggregates_parquet.py]: Does this really need 40 nodes?

File: dags/telemetry_aggregates_parquet.py (hunk 1 above)
Comment (robhudson) [reply dags/telemetry_aggregates_parquet.py]: I'm not sure? Should we? How does one do that?

File: dags/telemetry_aggregates_parquet.py (hunk 2 above)
Comment (robhudson) [reply dags/telemetry_aggregates_parquet.py]: I copied what the telemetry aggregates needs. It does seem high but I don't know how to judge how many nodes is really needed.

Comment (fbertsch) [reply dags/telemetry_aggregates_parquet.py]: Those are the release aggregates. I would imagine this doesn't need more than 5 nodes, can you confirm?

File: dags/telemetry_aggregates_parquet.py (hunk 1 above)
Comment (fbertsch) [reply dags/telemetry_aggregates_parquet.py]: The Longitudinal job does that, check how it's done there.

File: dags/telemetry_aggregates_parquet.py (hunk 2 above)
Comment (robhudson) [reply dags/telemetry_aggregates_parquet.py]: When I was testing it I was using a 5 node cluster on ATMO and less a few times and it ran fine.

File: dags/telemetry_aggregates_parquet.py (hunk 1 above)
Comment (robhudson) [reply dags/telemetry_aggregates_parquet.py]: Cool, I'll check it out and switch to that.

Comment (acmiyaguchi) [reply dags/telemetry_aggregates_parquet.py]: Only jobs that run via `mozetl_envvar` and `tbv_envvar` and the corresponding runners are supported via Databricks now. Custom scripts won't work without modifying the operator. Sorry for the lack of documentation on this behavior, it should be a bit more explicit.

**Comment by robhudson** [comment]
@fbertsch This has now been tested and updated and rebased.

**Pull request merged**